packages/sui-studio/src/components/layout/_style.scss

```diff
@@ -11,43 +10,49 @@
   &-sidebar {
     @include breakpoint-from(s) {
       flex: 0 0 $w-sidebar;
-      transform: translate3d(0, 0, 0);
+      // transform: translate3d(0, 0, 0);
+      width: $w-sidebar;
     }
-
+    top: $h-navHeader + 4;
     background-color: $c-white;
     border-right: 1px solid $c-gray-lightest;
     box-sizing: border-box;
     height: 100vh;
     overflow-x: hidden;
     overflow-y: scroll;
     position: fixed;
-    transform: translate3d(-100%, 0, 0);
+    transform: translate3d(0, 0, 0);
     transition: transform 0.5s ease-in-out;
-    z-index: 9998;
-    width: $w-sidebar;
+    z-index: 50;
 
     &Body {
       width: $w-sidebar;
       transition: width 0.25s ease-out;
     }
 
     &--open {
-      display: block;
-      transform: translate3d(0, 0, 0);
+      transform: translate3d(-100%, 0, 0);
     }
   }
 
   &-main {
     background-color: $bgc-main;
+    position: relative;
+    top: $h-navHeader + 4;
```

Comment:
this same value `$h-navHeader + 4` is used at least twice so perhaps it's time to assign it its own variable?

Reply:
✅ done